webapp/src/css/inbox.less

```diff
@@ -804,6 +813,18 @@ a.fa:hover {
       &:first-child {
         margin-top: 0;
       }
+
+      .message-error {
+        color: @failed-state-color;
+        font-size: @font-small;
+
+        &:before {
+          font-family: FontAwesome;
+          margin-right: 5px;
+          vertical-align: baseline;
+          content: @failed-state-icon;
+        }
+      }
```

Comment:
This is duplicated in the admin less file. Instead I think it would make sense to extract the common elements into a less file that both webapp and admin can include. I imagine there might be other message styles that are used in both which could be extracted too.

Reply:
Actually this was the only truly duplicated thing (which I did pull out into it's own file): Admin uses state icons, webapp uses state colors. Is there anything in particular you thought about including in this common messages LESS?